src/components/map/components/map/index.js

```diff
@@ -149,13 +183,13 @@ export class VlMap extends vlElement(HTMLElement) {
   get _projection() {
     return new OlProjection({
       code: "EPSG:31370",
-      extent: this._extent,
+      extent: [17736.0314, 23697.0977, 297289.9391, 245375.4223],
```

Comment:
Hier ook misschien constante van maken en in de naam zetten wat dit juist inhoudt.

Reply:
Done.